app/schemas/chat.py

```
@@ -0,0 +1,25 @@
+"""Chat schemas for request/response models."""
+
+from typing import List, Optional
+from pydantic import BaseModel
+
+
+class ChatMessage(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)